site/src/pages/RoadmapPage/ImportTranscriptPopup.tsx

```diff
@@ -214,7 +215,7 @@ const ImportTranscriptPopup: FC = () => {
         className="ppc-btn add-transcript-btn"
         onClick={() => setShowModal(true)}
       >
-        <FileEarmarkText className="add-transcript-icon" />
+        <DescriptionIcon style={{ fontSize: 20, marginRight: '5px' }} />
```

Comment:
same as other comments

you might consider `.ppc-btn .muiIconClassName { font-size: 20px; margin-right: 5px; } `